scripts/demo-annoy.py

```diff
@@ -4,42 +4,29 @@
 come with the base install of simsity.
 """
 
-from simsity.service import Service
-from simsity.datasets import fetch_clinc
-from simsity.indexer import PyNNDescentIndexer
-from simsity.preprocessing import ColumnLister
+import pandas as pd
 
 from sklearn.pipeline import make_pipeline
 from sklearn.feature_extraction.text import CountVectorizer
 from sklearn.decomposition import TruncatedSVD
 
+from simsity.service import Service
+from simsity.indexer import AnnoyIndexer
+
+df = pd.read_csv("tests/data/clinc-data.csv").head(100)
+
 # The encoder defines how we encode the data going in.
-encoder = make_pipeline(
-    ColumnLister(column="text"), CountVectorizer(), TruncatedSVD(n_components=10)
-)
+encoder = make_pipeline(CountVectorizer(), TruncatedSVD())
+encoder.fit(df["text"].tolist())
```

Comment:
Is there a reason for `.to_list()`? Should not `df["text"]` be enough?

Reply:
Not really. This PR is still a work in progress.